### DISCUSSION TITLE
Welcome to ICUAS '22 UAV Competition Discussions

### DISCUSSION THREAD

**Discussion by fpetric** [larics/icuas22_competition#8] [Announcements]
## Welcome to ICUAS '22 UAV Competition Discussions

Dear ICUAS UAV Competition teams, we will be using Discussions to manage your feedback, answer your questions, provide clarifications on the rules, provide additional technical details and so on.

Discussions do not replace the official competition e-mail, you can still use it but we suggest using Issues and Discussions, we are expecting to be more responsive on this platform. 

## ICUAS '22 UAV Competition organizing team

Next, the people behind this repo and the competition are also the people that you can tag for questions or discussion related to the following topics:

- @lmark1 -UAV control, system architecture, low level control, interfaces
- @aivano - world, arena layout and trajectory planning
- @AnaBatinovic - exploration
- @RobertMilijas - alvar tags detection and localization
- @fpetric - rules and everything else that you can think of

## Announcing updates

All new updates to the repo will be announced here. Currently, we are on version 1.0.2, with the most recent update being new world files with updated lighting. More details can be found in changelog.

## Thats' it for now! 

We hope that the installation went smoothly, you are all up and running and ready to start developing your solutions! And now, let's crash some simulated UAVs until we get it right! Ask away!

**Comment by fse5947** [comment]
@lmark1 are we restricted to delivering a single node solution or can we deliver a solution consisting of multiple nodes?

**Comment by fse5947** [comment]
@aivano is the AR tag always located on the back wall of the arena? Or could it be on the floor in zone 3 or on any other surface? Is there a limit to how high it can be located on the wall, with respect to the height of the arena?

**Comment by fse5947** [comment]
@aivano while the obstacles can be moved around, are they always pillars or can they be shaped differently, with potential dead-ends?

**Comment by fpetric** [comment]
To answer all questions @fse5947 (tagging @lmark1 and @aivano to notify them as well):
> @lmark1 are we restricted to delivering a single node solution or can we deliver a solution consisting of multiple nodes?

Your solution can have multiple nodes. Note that we do not plan to allow teams to use custom messages between nodes. 

> @aivano is the AR tag always located on the back wall of the arena? Or could it be on the floor in zone 3 or on any other surface? Is there a limit to how high it can be located on the wall, with respect to the height of the arena?

The AR tag can be located on any wall surface in the Zone 3 of the arena. Since camera is always facing forward, we haven't considered putting the tag on the floor. With respect to height, as of now there is no limit on the vertical position of the tag. 

> @aivano while the obstacles can be moved around, are they always pillars or can they be shaped differently, with potential dead-ends?

In the simulation, the obstacles will always be pillars. There may be dead ends, especially when considering the bounding box of the UAV (i.e. while there may be open space between pillars, the UAV may not be able to pass through).

Also, for clarity, these types of questions should be posted in the [Q&A section](https://github.com/larics/icuas22_competition/discussions/categories/q-a).
